<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunParameter</class>
 <widget class="QWidget" name="RunParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget_parameter">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>230</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWidget_parameterPage1">
      <attribute name="title">
       <string>实验选择</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_select_model">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VGG+SSD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VGG+DSSD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VGG+ResNet+SSD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SqueezeNet+SSD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SqueezeNet+DenseNet+SSD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>YOLO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SqueezeNet+YOLO</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>可选模型</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_model_info">
          <attribute name="title">
           <string>模型信息</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_video_info">
          <attribute name="title">
           <string>数据信息</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>结果参数</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>准确率</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_percision"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>召回率</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_recall"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>mAP</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_mAP"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>候选概率</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>目标覆盖率</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_prob"/>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_iou"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
